README.md

```diff
@@ -1,10 +1,13 @@
-## Docker for Visual Studio Code
 
-[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/Nightly/vscode-docker-nightly-2?branchName=master)](https://dev.azure.com/ms-azuretools/AzCode/_build/latest?definitionId=22&branchName=master)
+The Docker extension makes it easy to build, manage, and deploy containerized applications from Visual Studio Code. It also provides one-click debugging for Node.js, Python, and .NET Core.
+
+**Check out the [Working with containers](https://aka.ms/AA7arez) topic on the Visual Studio Code documentation site to get started**.
 
-The Docker extension makes it easy to build, manage and deploy containerized applications from Visual Studio Code.
+![Docker extension overview](resources/readme/overview.gif)
```

Comment:
I think the GIF covers the right content, but there's a couple changes I'd make to it--
1. Shrink the window and zoom in a bit
1. Record only the VSCode window, thus hiding your terrifying launch bar :smile:

Reply:
The lanuch bar is downright scary 😆

Reply:
LOL, good point. Will re-record.